### PR TITLE
Add notification timeout for "Accounts updated" msg. Fixes #229

### LINKED ISSUE #229: Repeated messages not disappearing after a new notification
Steps to reproduce:

1) Go to app.chronologic.network
2) Make sure you're logged in with Metamask
3) Change your wallet an arbitrary number of times
4) Request test DAY from faucet
5) Voilá, you get this:

image

Those notifications don't go away unless you click them away.

### PR DESCRIPTION
**Pull request by kennym**

### CODE REVIEW

File: app/components/Common/MetamaskComponent.js [hunk 1]
```diff
@@ -103,4 +103,4 @@ MetamaskComponent.propTypes = {
   history: PropTypes.any,
 };
 
-export default MetamaskComponent;
\ No newline at end of file
+export default MetamaskComponent;
```
Comment (Bagaric) [inline app/components/Common/MetamaskComponent.js]: This line doesn't need to be changed. Please revert this if possible so we don't merge unnecessary changes.

File: app/components/Common/MetamaskComponent.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ class MetamaskComponent extends Component {
     const accountsChanged = this.state.accounts.length !== web3Service.accounts.length || (this.state.accounts.length > 0 && this.state.accounts[0] !== web3Service.accounts[0]);
     if (accountsChanged) {
       this.setState({ accounts: web3Service.accounts });
-      showNotification(`Accounts updated`, 'info');
+      showNotification(`Accounts updated`, 'info', 15000);
```
[the remark this answers is not in the document]
Comment (kennym) [reply app/components/Common/MetamaskComponent.js]: @Bagaric done! Thank you

File: app/components/Common/MetamaskComponent.js (hunk 1 above)
Comment (kennym) [reply app/components/Common/MetamaskComponent.js]: ok

**Comment by kennym** [comment]
I messed up my branch with rebasing - closing this and opening another PR.
